### PR TITLE
Add v0.build and vusercontent.net

### PR DESCRIPTION
**Pull request by MaxLeiter**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 
  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The submitter acknowledges that it is their responsibility to maintain the domains within their section. This includes removing names which are no longer used, retaining the _psl DNS entry, responding to e-mails to the supplied address. Failure to maintain entries may result in removal of individual entries or the entire section.

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x]  The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

[x] Description of Organization
====
I'm on the AI team at Vercel building v0.dev. It allows you to generate React components and chat with the v0 LLM. Outputs generated by the LLM can be shared and published on v0.dev to the v0.build subdomain. The vusercontent domain is where we host the user-submitted content. 

For both domains, we intend to add user-customizable subdomains similar to GitHub Pages.

Organization Website: 
vercel.com <-- parent company (already in the PSL)
v0.dev <-- this particular PR

[x] Reason for PSL Inclusion
====

- We want our users' sites/subdomains to be isolated for other customers (cookies, suffix highlighting, etc)

You can see our previous (accepted) request for vercel.app here: https://github.com/publicsuffix/list/pull/1019

Number of users this request is being made to serve:

[x] DNS Verification via dig
=======

```bash
> dig +short TXT _psl.vusercontent.net
"https://github.com/publicsuffix/list/pull/2121"

> dig +short TXT _psl.v0.build
"https://github.com/publicsuffix/list/pull/2121"
```

[x] Results of Syntax Checker (`make test`)
=========

```
Results of Syntax Checker (`make test`)
=========

============================================================================
Testsuite summary for libpsl 0.21.5
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
Making check in msvc
```

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -15596,6 +15596,8 @@ web.val.run
 vercel.app
 vercel.dev
 now.sh
+vusercontent.net
+v0.build
```
Comment (groundcat) [inline public_suffix_list.dat:15600]: Just a reminder to sort your entries according to the [guidelines](https://github.com/publicsuffix/list/wiki/Guidelines#sort-your-submission-correctly-important). Keep the list of suffixes (if there is more than one) in alphabetical order. Sort first by the TLD, then by the first label to the left of the TLD, and so on.

**Comment by wdhdev** [comment]
* [x] Expiration (Note: Must STAY >2y at all times)
  * [x] v0.build expires 2027-08-14
  * [ ] vusercontent.net expires 2024-09-09
* [x] DNS _psl entries (Note: Must STAY in place)
  * [ ] `_psl.v0.build`
  * [x] `_psl.vusercontent.net`
* [ ] Tests pass
* [ ] Sorting
* [x] Reasoning/Organization description
  * User is associated with Vercel, even though he has not modified this entry before.
    * Vercel email address used in commit 5543c405c819a89883c1c1dd14fa9100aa98d23a: max.leiter@vercel.com 
    * User is in the @vercel GitHub organisation
* [x] Non-personal email address

---

Notes:
* `vusercontent.net` expiry is not >2y
* No TXT record was found at `_psl.v0.build`
* You have not indicated if tests have passed
* You need to fix the sorting of these domains as mentioned by @groundcat

**Comment by cramforce** [comment]
Taking over from @MaxLeiter as he is out on vacation.

- Fixed `vusercontent.net` registration expiration
- Fixed TXT record  at `_psl.v0.build`
- Added indication of tests passing
- Fixed sorting

Thanks for the reviews!

**Comment by wdhdev** [comment]
Looks good to me. @simon-friedberger ready to merge.

**Pull request merged**